tests/e2e/test_image.py

```diff
@@ -442,3 +442,54 @@ def test_external_image_build(
     if f"Successfully built {img_uri_str}" not in result.stdout:
         LOGGER.warning(result.stdout)
         raise AssertionError("Successfully built message was not found.")
+
+
+@pytest.mark.skipif(
+    sys.platform == "win32", reason="docker is not installed on Windows nodes"
+)
```

Comment:
win machines do have Docker [installed](https://github.com/actions/virtual-environments/blob/main/images/win/Windows2022-Readme.md#tools), or we require docker-buildx?